src/api/spec/views/webui/user/show.html.erb_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
```

Comment:
Just transcribed to the proper type of spec... we can get rid of it if you prefer that